integreat_cms/cms/models/contact/contact.py

```diff
@@ -63,19 +63,27 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the contact
         """
-        point_of_contact_for = (
-            f"{self.point_of_contact_for.strip()}: "
-            if self.point_of_contact_for and self.point_of_contact_for.strip()
-            else ""
-        )
-        name = f"{self.name.strip()} " if self.name and self.name.strip() else ""
-        details = [
-            detail
-            for detail in [self.email, self.phone_number, self.website]
-            if detail
-        ]
-        details_str = f"({', '.join(details)})" if details else ""
-        return f"{point_of_contact_for}{name}{details_str}"
+        location_name = str(self.location)
+        additional_attribute = self.get_additional_attribute()
+        return _("%s with %s") % (location_name, additional_attribute)
+
+    def get_additional_attribute(self) -> str:
+        if self.point_of_contact_for:
+            return _("point of contact for: %s") % self.point_of_contact_for
+
+        if self.name:
+            return _("name: %s") % self.name
+
+        if self.email:
+            return _("email: %s") % self.email
+
+        if self.phone_number:
+            return _("phone number: %s") % self.phone_number
+
+        if self.website:
+            return _("website: %s") % self.website
+
+        return ""
```

Comment:
This might make the message show `Contact for [NAME_OF_LOCATION] with  was saved` when this case is hit.
Though it *is* a degenerate case that we currently don't anticipate in the system, as we constrain the model to have at least one field set. So maybe it's fine to just leave it, but on the other hand dropping the ` with ` part seems easy enough

Reply:
I removed the with from the String now :)

Reply:
Tiny nitpick, if the detail happens to be empty, the result of `__str__()` has trailing whitespace. I'm unsure how to account for that cleanly and without cluttering the code *(using `return _(" with name: %s") % self.name` would put it in the translations where it does not belong, but I'm not sure shenanigans like `return f' {_("with name: %s") % self.name}` is much better either)*

Reply:
Do you have a preference out of these three options?
1. Leave it as it is, as it's only a theoretical possibility and since the detail isn't allowed to be empty anyways would never happen in reality
2. But it in the translation (which is not really clean)
3. Do stuff like return f' {_("with name: %s") % self.name}

Reply:
(for now I changed it to option 2)

Reply:
Personally, I'm itching to do this *(here trying to not have an extra f string in every if branch)* but you guys need to decide whether that's too much visual clutter for one space *(or alternatively the `" ".join()` solution)*
```python
    def get_additional_attribute(self) -> str:
        string = None

        if self.point_of_contact_for:
            string = _("with point of contact for: {}").format(self.point_of_contact_for)

        if self.name:
            s = _("with name: {}").format(self.name)

        if self.email:
            s = _("with email: {}").format(self.email)

        if self.phone_number:
            s = _("with phone number: {}").format(self.phone_number)

        if self.website:
            s = _("with website: {}").format(self.website)

        if s:
            return f" {s}"

        return ""
```

Reply:
I like this solution